04-ruby-mvc-sinatra/sinatra-rest/readme.md

```diff
@@ -12,24 +12,43 @@
 - Use a global template - layout.erb
 - Use forms to submit POST and GET requests
 
+## Opening Framing
+We've learned about databases using SQL. We've also learned about objects in ruby. But we haven't seen any HTML since Unit 1! What gives? How are we going to display all of the things in our database? How can we create a UI to execute CRUD functionality on our different ruby objects? Sinatra. Today you'll be learning about how to create routes and views. In order to talk about how Sinatra functions, we need to talk about how HTTP requests work. Let's talk about REST.
+
```

Comment:
@adambray should have a diagram from his lesson about how all the pieces fit together in the big picture. maybe reshow here?